net-libs/telepathy-qt/telepathy-qt-0.9.8.ebuild

```diff
@@ -0,0 +1,62 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{5,6,7} )
+inherit python-any-r1 cmake-utils virtualx
+
+DESCRIPTION="Qt bindings for the Telepathy D-Bus protocol"
+HOMEPAGE="https://telepathy.freedesktop.org/"
+SRC_URI="https://telepathy.freedesktop.org/releases/${PN}/${P}.tar.gz"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+IUSE="debug farstream test"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+RDEPEND="
+	dev-qt/qtcore:5
+	dev-qt/qtdbus:5
+	dev-qt/qtgui:5
+	dev-qt/qtnetwork:5
+	dev-qt/qtxml:5
+	farstream? (
+		>=net-libs/telepathy-farstream-0.2.2
+		>=net-libs/telepathy-glib-0.18.0
+	)
+	test? (
+		dev-libs/dbus-glib
+		dev-libs/glib:2
+		dev-qt/qttest:5
+		dev-python/dbus-python[${PYTHON_USEDEP}]
```

Comment:
`${PYTHON_USEDEP}` exists in `python-single-r1`, but not in `python-any-r1`. Look here for what you need to do: https://wiki.gentoo.org/wiki/Project:Python/python-any-r1#Package_with_additional_dependencies

Reply:
Meh, I copy-pasted the code from the link and didn't notice the unpaired parenthesis. @a17r would you mind to fix the example in wiki? I For some reason I'm unable to sign in there. :thinking:

Reply:
thanks, it is fixed now ;)